modules/items/client.lua

```diff
@@ -22,6 +22,17 @@ local ox_inventory = exports[ox.resource]
 -----------------------------------------------------------------------------------------------
 -- Clientside item use functions
 -----------------------------------------------------------------------------------------------
+for key, value in pairs(Items) do
```

Comment:
🔕 This won't do, though I do have a good solution for it. Remove this change and squash, then I'll add another change on top for handling notifications.

The notification stuff should be an improvement for all items to be able to utilise, rather than reducing the number of items that need to be explicitly registered. There should already be something in place that sets these items as usable and will fallback to a generic use event without a callback (I'll double check).

Reply:
👍 Reverted everything in `items/client.lua` back to normal